### PR TITLE
This pull request adds alt text to the retrolab readme

### PR DESCRIPTION
**Pull request by tonyfast**
This pull request is the outcome of our accessibility workshop. It adds alt to the current retrolab readme. Thanks to @isabela-pf @jweill-aws for all the work.

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks all!

File: app/package.json [hunk 1]
```diff
@@ -135,7 +135,7 @@
     "glob": "~7.1.6",
     "mini-css-extract-plugin": "~0.9.0",
     "npm-run-all": "^4.1.5",
-    "playwright": "^1.12.3",
+    "playwright": "^1.17.1",
```
Comment (jtpio) [inline app/package.json:138]: Looks like this PR might be bringing some additional changes?

Comment (tonyfast) [reply app/package.json:138]: oh i think i messed something up merging. ill have a look over the weekend. ill fix this soon.

**Comment by jtpio** [comment]
@tonyfast I rebased this PR to only include the change to `README.md`.

However I'm not able to push to the fork repo, probably because the PR was opened from the `main` branch.

Here is the link to the diff if you would like to open a new PR with the single commit: https://github.com/jupyterlab/retrolab/compare/main...jtpio:readme-a11y?expand=1

Thanks!

**Comment by jtpio** [comment]
Closing as fixed by #310.

**Pull request closed without merging**